src/fmeval/eval_algorithms/context_quality.py

```diff
@@ -0,0 +1,291 @@
+from typing import Optional, List, Dict, Any
+import logging
+from dataclasses import dataclass
+
+from fmeval.constants import (
+    DatasetColumns,
+    MEAN,
+)
+from fmeval.data_loaders.data_config import DataConfig
+from fmeval.data_loaders.util import get_dataset
+from fmeval.eval_algorithms import EvalAlgorithm, EvalScore, EvalOutput
+from fmeval.eval_algorithms.common import evaluate_dataset
+from fmeval.eval_algorithms.eval_algorithm import (
+    EvalAlgorithmInterface,
+    EvalAlgorithmConfig,
+)
+from fmeval.eval_algorithms.save_strategy import SaveStrategy
+from fmeval.eval_algorithms.util import get_dataset_configs, validate_dataset
+from fmeval.model_runners.bedrock_model_runner import BedrockModelRunner
+from fmeval.model_runners.model_runner import ModelRunner
+from fmeval.transforms.transform import Transform
+from fmeval.transforms.util import validate_call
+from fmeval.model_runners.composers.composers import PromptComposer
+from fmeval.transforms.transform_pipeline import TransformPipeline
+from fmeval.util import get_eval_results_path
+
+logger = logging.getLogger(__name__)
+
+CONTEXT_PRECISION_SCORE = "context_precision_score"
+
+DEFAULT_CONTEXT_PRECISION_PROMPT_TEMPLATE = (
+    "Given question, answer and context verify if the context was useful in "
```

Comment:
Nit: lack of commas make this confusing to read. Can you please change it to "Given a question, answer, and context, verify ..."